<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="dis_name">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;고사장 입력&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_name">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>서울</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>경기</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>인천</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>부산</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>대구</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>대전</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>세종</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>광주</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>경남</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>경북</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>울산</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>전남</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>전북</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>충청</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>강원</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>제주</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="push_alt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>확인</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_name">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="push_name">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QDial" name="dial_input">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>150</y>
      <width>121</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_dial">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
